action.yml

```diff
@@ -82,7 +82,10 @@ runs:
 
     - name: Update Python Versions
       shell: bash
-      run: "[[ \"${{inputs.update_python}}\" == \"true\" ]] && sed -i 's/python_versions:.*/python_versions: [\"3.8\", \"3.9\", \"3.10\", \"3.11\"] /' \"${{ inputs.action_path }}\""
+      run: |
+        if [ "${{inputs.update_python}}" == "true" ]; \
+        then sed -i 's/python_versions:.*/python_versions: [3.8, 3.9, 3.10, 3.11] /' "${{ inputs.action_path }}"; \
```

Comment:
Please put the versions in quotes, otherwise yaml can interpret them as numbers which can turn "3.10" into "3.1".